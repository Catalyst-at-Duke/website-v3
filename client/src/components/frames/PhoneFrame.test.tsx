import React from "react";
import { shallow, configure } from "enzyme";
import { PhoneFrame } from ".";

describe("PhoneFrame with no props", () => {
  const phone = shallow(<PhoneFrame />);
  it("should match the snapshot", () => {
    expect(phone.html()).toMatchSnapshot();
  });
});
