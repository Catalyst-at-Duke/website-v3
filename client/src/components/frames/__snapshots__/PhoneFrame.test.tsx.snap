// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhoneFrame with no props should match the snapshot 1`] = `"<div style=\\"display:flex;flex-direction:column\\"><div style=\\"display:flex;justify-content:flex-end\\"><div class=\\"hbutton\\" style=\\"background-color:#555555\\"></div></div><div style=\\"display:flex;flex-direction:row\\"><div class=\\"phone-outer\\" style=\\"background-color:#26272C\\"><div class=\\"phone-inner\\" style=\\"background:url(tech_types.png);background-position:center\\"><div class=\\"app-box\\"><div class=\\"app-icon\\" style=\\"background-color:#ffffff\\"></div><div class=\\"app-icon\\" style=\\"background-color:#ffffff\\"></div><div class=\\"app-icon\\" style=\\"background-color:#ffffff\\"></div><div class=\\"app-icon\\" style=\\"background-color:#ffffff\\"></div><div class=\\"app-icon\\" style=\\"background-color:#ffffff\\"></div><div class=\\"app-icon\\" style=\\"background-color:#ffffff\\"></div><div class=\\"app-icon\\" style=\\"background-color:#ffffff\\"></div><div class=\\"app-icon\\" style=\\"background-color:#ffffff\\"></div></div><div class=\\"flex-box\\" style=\\"position:relative;bottom:0\\"><div class=\\"circle\\" style=\\"background-color:#C4C4C4\\"></div><div class=\\"circle\\" style=\\"background-color:#ffffff\\"></div></div></div></div><div style=\\"display:flex;flex-direction:column\\"><div class=\\"vbutton\\" style=\\"background-color:#555555;margin-top:40px\\"></div><div class=\\"vbutton\\" style=\\"background-color:#555555;margin-top:1px\\"></div></div></div></div>"`;
